packages/dds/tree/src/simple-tree/prepareForInsertion.ts

```diff
@@ -28,12 +28,14 @@ import {
 	tryUnhydratedFlexTreeNode,
 	unhydratedFlexTreeNodeToTreeNode,
 } from "./core/index.js";
-import { debugAssert } from "@fluidframework/core-utils/internal";
+import { debugAssert, oob } from "@fluidframework/core-utils/internal";
 
 /**
  * Prepare content from a user for insertion into a tree.
  * @remarks
  * This validates and converts the input, and if necessary invokes {@link prepareContentForHydration}.
+ *
+ * The next edit made to the context must be the insertion of this content otherwise hydration will break.
```

Comment:
Nit
```suggestion
 * The next edit made to the context must be the insertion of this content, otherwise hydration will break.
```

Reply:
Is there a specific API we should link to for the "next edit made" note?

Reply:
should be more clear now